docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -42,6 +42,8 @@ It's that simple.  This will start the Metabase application using all of the def
 
 At this point your ready to go!  You can access your new Metabase server on port 3000, most likely at [localhost:3000](http://localhost:3000)
 
+You can use another port than 3000 by setting the `MB_JETTY_PORT` environment variable before running the jar
```

Comment:
How do you do that?

Reply:
How is MB_JETTY_PORT variable added please. am running windows.

Reply:
In windows environment use SET MB_JETTY_PORT=12345